proto/ibc/core/connection/v1/tx.proto

```diff
@@ -67,6 +67,8 @@ message MsgConnectionOpenTry {
   ibc.core.client.v1.Height consensus_height = 11
       [(gogoproto.moretags) = "yaml:\"consensus_height\"", (gogoproto.nullable) = false];
   string signer = 12;
+  // extra data for state machaines that can't introspect their own consensus state.
+  bytes host_consensus_state_info = 13;
```

Comment:
To match the [spec changes that were merged](https://github.com/cosmos/ibc/pull/839)...

```suggestion
  bytes host_consensus_state_proof = 13;
```